telegram_export/downloader.py

```diff
@@ -254,6 +267,9 @@ def _get_name(self, peer_id):
         if not ext:
             ext = export_utils.get_extension(media_row[4])
 
+        if isinstance(filename, str):
+            filename = export_utils.format_filename(filename)
```

Comment:
please let me know how to do this

Reply:
Making a new branch is a bit harder because you have to revert the changes you have made in `master` first. Easiest way for you is:
1. Wait until `MAX_SIZE` bug fix is merged.
2. Delete your fork.
3. Fork again.
4. Make new changes.

But you can look up online how to do it without deleting the fork if you want.

Reply:
https://github.com/expectocode/telegram-export/pull/99#discussion_r240002438